tests/core/fs/workspacefs/test_sync_transactions.py

```diff
@@ -192,7 +192,7 @@ def test_merge_folder_manifests_with_concurrent_remote_change(
         .to_remote(author=my_device, timestamp=timestamp)
     )
 
-    prevent_sync_pattern = re.compile(r".*\.tmp\Z")
+    prevent_sync_pattern = Regex.from_regex_str(r".*\.tmp\z")
```

Comment:
Why the regex expression have changed ?

Reply:
Rust's regex does not recognize capital `Z` but only `z`. The `py_to_rs_regex` function does this for us but as we don't need it anymore I've changed the regex.

Reply:
ok